pem/lib/pem/options.rb

```diff
@@ -18,9 +18,16 @@ def self.available_options
                                      description: "Generate a p12 file additionally to a PEM file",
                                      is_string: false,
                                      default_value: true),
+        FastlaneCore::ConfigItem.new(key: :current_active_days_limit,
```

Comment:
Can we change this key name to be something like `:active_days_limit` or something else that is a bit more self-explanatory?